GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -50,14 +59,23 @@ public sealed class GitStatusMonitor : IDisposable
         // Timestamps to schedule status updates, limit the update interval dynamically
         // Note that TickCount wraps after 25 days uptime, always compare diff
 
-        // Next scheduled update time
+        /// <summary>
+        /// Next scheduled update time
+        /// </summary>
         private int _nextUpdateTime;
 
-        // Earliest time for an scheduled update (interactive requests bypasses this)
+        /// <summary>
+        /// Earliest time for an scheduled update (interactive requests bypasses this)
+        /// </summary>
         private int _nextEarliestTime;
 
         private GitStatusMonitorState _currentStatus;
 
+        /// <summary>
+        /// git-status command is running that is not cancelled
+        /// </summary>
+        private bool _uncancelledCommandIsRunning;
```

Comment:
"uncancelled" this looks very odd, ditto for "is running that is not cancelled" - this sounds like an oxymoron...

Reply:
> "is running that is not cancelled" - this sounds like an oxymoron...

In a few more words: git-statis is started and the status is nether completed (with success or error) or cancelled explicitly. If false, a new git-status can be started.
I do not have a better description. the variably name can be composed from the words in the description too.

Reply:
Why the original name didn't work?

Reply:
The only clear meaning of a `_commandIsRunning` would be:
```
    try
    {
         _commandIsRunning = true;
         RunTheCommand()
    }
    finally
    {
        _commandIsRunning = false;
    }
```
without any conditions.
If it deviates from this scheme, it should be reflected in the identifier - not hidden in the docs.
It should be clear, why this flag must not be reset in case of cancellation. Cancelled commands are detached and ignored. We don't want to wait for their cancellation and start the next command.
`_commandIsRunningAndUncancelled` might be clearer.

Reply:
It is not a change. It is a clarification and completes the handling.

Reply:
"Uncancelled" is triggering me :) It's so Unenglish (pun intended).
How about `_commandRunningWithoutCancellation` or `_commandIsRunningAndNotCancelled`?